modules/lvms-installing-logical-volume-manager-operator-using-cli.adoc

```diff
@@ -4,22 +4,20 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="install-lvms-operator-cli_{context}"]
-= Installing {lvms} with the CLI
+= Installing {lvms} by using the CLI
 
-As a cluster administrator, you can install {lvms-first} by using the CLI.
+As a cluster administrator, you can install {lvms} by using the OpenShift CLI.
 
 .Prerequisites
 
 * You have installed the OpenShift CLI (`oc`).
-
-* You have logged in as a user with `cluster-admin` privileges.
+* You have logged in to {product-title} as a user with `cluster-admin` and Operator installation permissions.
 
 .Procedure
 
-. Create a namespace for the {lvms} Operator.
-
-.. Save the following YAML in the `lvms-namespace.yaml` file:
+. Create a `Namespace` custom resource (CR) YAML file:
```

Comment:
Namespace is not a custom resource. We can refer to it as just `Namespace`.

Reply:
But the YAML configuration/format for `Namespace` looks similar to any other custom resource. 
For example: The `Namespace` also contains the `apiVersion: v1` and `kind: Namespace` fields. 

Is it not a CR for creating a namespace?

Reply:
Still, a `Namespace` is a built-in resource in Kubernetes, just like `Pod`, `Deployment`, `ConfigMap`, etc. You can read the difference [here](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).

Reply:
Updated as per the suggestion.